bindings/wasm/examples-account/src/lazy.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { AccountBuilder, ExplorerUrl } from './../../node/identity_wasm.js';
+
+async function lazy() {
+
+    // Create a new Account with auto publishing set to false.
+    // This means updates are not pushed to the tangle automatically.
+    // Rather, when we publish, multiple updates are batched together.
+    let builder = new AccountBuilder({
+        autopublish: false
+    });
+    let account = await builder.createIdentity();
+
+    // Add a new service to the local DID document.
+    await account.createService({
+        fragment: "example-service",
+        serviceType: "LinkedDomains",
+        endpoint: "https://example.org"
+    })
+
+    // Publish the newly created DID document,
+    // including the new service, to the tangle.
+    await account.publish();
+
+    // Add another service.
+    await account.createService({
+        fragment: "another-service",
+        serviceType: "LinkedDomains",
+        endpoint: "https://example.org"
+    });
+
+    // Delete the previously added service.
+    await account.deleteService("example-service");
+
+    // Publish the updates as one message to the tangle.
+    // Also see: account.publishWithOptions(..)
+    await account.publish();
```

Comment:
I think it's good to give a hint about the existence of the other method, since there's no example for it.

Reply:
It's not mentioned in Rust and I think `publish` and `publishWithOptions` should be combined in the Wasm bindings since similar `*with_options` methods have been combined in other places already since it's easier to omit parameters in Javascript.

Reply:
I merged both methods.